src/assets/scripts/client/InputController.js

```diff
@@ -833,18 +721,18 @@ export default class InputController {
     /**
      * @for InputController
      * @method processTransmitCommand
-     * @param commandParser {CommandParser}
+     * @param AircraftCommandParser {AircraftCommandParser}
      * @return {boolean}
      */
-    processTransmitCommand(commandParser) {
+    processTransmitCommand(AircraftCommandParser) {
```

Comment:
this should be lowercase since it is an instance

Reply:
Done